DataAccess/UnitOfWork/IUnitOfWork.cs

```diff
@@ -0,0 +1,16 @@
+﻿using DataAccess.Repository;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccess.UnitOfWork
+{
+    public interface IUnitOfWork
```

Comment:
I dont recommend use it. Using IRepository is enough for abstraction. Delete it please